bin/java

```diff
@@ -266,13 +266,13 @@ _install_metrics_agent() {
   local ctxDir=${1:-BUILD_DIR}
   local curDir=${JVM_COMMON_DIR:-$(cd $(dirname ${BASH_SOURCE[0]}) && cd .. && pwd )}
   local binDir="${ctxDir}/.heroku/bin/"
-  local agent_jar="${binDir}/heroku-metrics-agent.jar"
+  local agent_jar="${binDir}/heroku-jvm-metrics-agent.jar"
 
   mkdir -p ${binDir}
   curl --retry 3 -s -o ${agent_jar} \
       -L ${HEROKU_METRICS_JAR_URL:-"https://repo1.maven.org/maven2/com/heroku/agent/heroku-java-metrics-agent/3.7/heroku-java-metrics-agent-3.7.jar"}
   [ ! -f ${agent_jar} ] && warning_inline "failed to install metrics agent!"
 
   mkdir -p ${ctxDir}/.profile.d
-  cp $curDir/opt/heroku-metrics-daemon.sh $ctxDir/.profile.d/
```

Comment:
Suggestion: Because this script no longer runs a daemon, drop that from the name. Maybe something like: `heroku-jvm-metrics-options.sh` since it sets options/flags for the jvm?

Reply:
truth. #57